.github/labeler.yml

```diff
@@ -6,20 +6,16 @@ Cython / Python:
   - 'python/**'
   - 'notebooks/**'
   
-libcuml:
+CUDA/C++:
```

Comment:
@JohnZed @dantegd Would this label be correct?

Reply:
@JohnZed @dantegd feel free to add more labels/paths if needed

Reply:
This looks good.